diagnostics/src/daq_trace/mod.rs

```diff
@@ -300,7 +312,7 @@ async fn poll_kafka_msg(consumer: StreamConsumer, common_dig_data_map: Digitiser
                     }
                 }
 
-                consumer.commit_message(&msg, CommitMode::Async).unwrap();
+                let _ = consumer.commit_message(&msg, CommitMode::Async);
```

Comment:
This should probably emit an error trace:
```rust
if let Err(e) = consumer.commit_message(&msg, CommitMode::Async) {
    error!("Failed to commit message consume: {e}");}
```

Reply:
Not needed here, since this is a TUI app that will 1) never be connected to an external trace collection tool and, 2) never show log messages on the console.